pyanaconda/ui/gui/spokes/root_password.py

```diff
@@ -339,8 +346,5 @@ def on_lock_clicked(self, lock):
                 # if the checkbox has not yet been manipulated by the user
                 # we can ignore this run and not adjust the fields
                 return True
-        self.password_entry.set_sensitive(not lock.get_active())
```

Comment:
Is it necessary to remove this code ? I though we disabled the root password locking checkbox on RHEL so I would assume this callback to be never called (or only during refresh & that should be interrupted by the check).

Reply:
The removal of `grab_focus` below was overzealous, but the `set_sensitive` must go for this to work.